lib/utils/Devirt.cpp

```diff
@@ -374,16 +374,19 @@ Devirtualize::makeDirectCall (CallSite & CS) {
 
   std::vector<const Function*> Targets;
 
-  if (CTF->size(CS) && CTF->isComplete(CS)) {
-    // TODO should we allow non-matching targets?
-    // TODO non-matching targets leads to crashes in bounce creation
-    // TODO formerly, all call-target-finder tarets were included:
-    //   Targets.insert (Targets.begin(), CTF->begin(CS), CTF->end(CS));
-    // TODO presently we filter out unmatching targets:
-    for (auto F = CTF->begin(CS); F != CTF->end(CS); ++F)
+  //if (CTF->size(CS) && CTF->isComplete(CS)) {
+  //  // TODO should we allow non-matching targets?
+  //  // TODO non-matching targets leads to crashes in bounce creation
+  //  // TODO formerly, all call-target-finder tarets were included:
+  //  //   Targets.insert (Targets.begin(), CTF->begin(CS), CTF->end(CS));
+  //  // TODO presently we filter out unmatching targets:
+  //  for (auto F = CTF->begin(CS); F != CTF->end(CS); ++F)
```

Comment:
Again, shouldn't we completely erase this?

Reply:
Done.